tools/dreport.d/plugins.d/bmcweb

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# config: 123 20
+# @brief: Get bmcweb session details
+#
+
+. "$DREPORT_INCLUDE/functions"
+
+# collect data only if bmcweb is present
```

Comment:
any specific reason?

Reply:
> any specific reason?

This file aid in debug for certain field issues by collecting information on the open sessions and event subscribers for bmcweb

Reply:
Sorry for the confusion why there is a check to make sure the bmcweb application is installed or not?  Won't we support the Redfish in certain OpenBMC based OpenPOWER systems?

Reply:
this check is done in general as with bitbake we have the capability to include/not include a pacakge or start/don't start a service.